test/python/test_proof_trace_slow.py

```diff
@@ -37,14 +53,20 @@ def test_file(self):
 
            # check that the first event is the rewrite a() => b()
            self.assertTrue(trace.trace[0].is_step_event())
-           self.assertEqual(trace.trace[0].step_event.rule_ordinal, 95)
+           rule_ordinal = trace.trace[0].step_event.rule_ordinal
+           axiom = definition.get_axiom_by_ordinal(rule_ordinal).__repr__()
```

Comment:
```suggestion
           axiom = repr(definition.get_axiom_by_ordinal(rule_ordinal))
```
and similarly elsewhere.